compiler_opt/tools/combine_tfa_policies_lib.py

```diff
@@ -0,0 +1,138 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Combines two tf-agent policies with the given state and action spec."""
+from typing import Dict, Optional, Tuple
+
+import tensorflow as tf
+import hashlib
+
+import tf_agents
+from tf_agents.trajectories import time_step as ts
+from tf_agents.typing import types
+from tf_agents.trajectories import policy_step
+import tensorflow_probability as tfp
+
+
+class CombinedTFPolicy(tf_agents.policies.TFPolicy):
+  """Policy which combines two target policies."""
+
+  def __init__(self, *args, tf_policies: Dict[str, tf_agents.policies.TFPolicy],
+               **kwargs):
```

Comment:
do we need the `kwargs` right now? let's not have it if not (we can always add later)

Reply:
I am just using kwargs and args to pass these to the TFPolicy constructor so if they are needed they would be needed by TFPolicy. I am not completely sure on that last part though.

Reply:
I understand that, my point is that we don't currently make use of it. So let's remove them, and if we ever need to add them back, that's easy to do.

Reply:
It's used to pass time_step_spec. I am keeping it.